src/components/questions-modal/questions-modal.html

```diff
@@ -10,7 +10,7 @@
 </ion-header>
 
 <ion-content padding>
-  <ion-row *ngFor="let question of tellMeQuestions; let i = index" class="mes-text-md align-items-center mes-bottom-border">
+  <ion-row *ngFor="let question of options; let i = index" class="mes-text-md align-items-center mes-bottom-border">
```

Comment:
Is the `let i = index` needed?

Reply:
is needed, it is padded to toggle option few lines below